react/Header/Header.less

```diff
@@ -153,3 +154,9 @@
     display: inline-block;
   }
 }
+
+.bellWrapper {
+  display: inline-block;
+  margin-top: 15px;
```

Comment:
Is this margin still necessary even with the flex centering above?

Reply:
Good point, added that margin before adding the vertical centering. Removed.